src/xcode/ENA/ENA/Source/Views/Home Screen/Cells/ActivateCollectionViewCell.swift

```diff
@@ -29,8 +31,14 @@ class ActivateCollectionViewCell: HomeCardCollectionViewCell {
 		accessibilityTraits = .button
 	}
 
-	func configure(title: String, icon: UIImage?, animationImages: [UIImage]? = nil, accessibilityIdentifier: String) {
+	func configure(title: String, subtitle: String? = nil, icon: UIImage?, animationImages: [UIImage]? = nil, accessibilityIdentifier: String) {
 		self.titleLabel.text = title
+		self.subtitleLabel.text = subtitle
+		if self.subtitleLabel.isTruncated {
+			self.subtitleLabel.text = AppStrings.ExposureNotificationSetting.euWideSubtitle
```

Comment:
remove all unneeded `self` from this method

Reply:
Done